modules/ROOT/pages/validate-4-task.adoc

```diff
@@ -1,34 +1,57 @@
-= Validate Query Parameters and Headers Using APIkit
+= Validating Query Parameters and Headers
```

Comment:
Check the guidelines for gerund/imperative headings. I believe this should be imperative.

Reply:
@hannanelson, thanks for your feedback!
I understand that we should use gerund in H1 and infinitive in H2+.
Actually, the APIkit beta site (https://dev-docs-internal.kqa.msap.io/beta-apikit-new-content-structure/apikit/4.x/overview) has been structured like that.
Please, can you confirm if gerund is OK for H1 and infinitive for H2+?
Thanks!

Reply:
Yes. If there are multiple tasks, the H1 should be gerund and the H2s imperative.

Reply:
@In this case, we have task 1 = H1 and task 2 = H2. So we can consider we have two tasks.
As per our style guide, we should leave the gerund in this case:
"Task pages introduced using a gerund title and narrative text about an action typically lead to one or more imperative-titled task subtopics (Build Your Database, Create a Message Flow) include numbered steps."